build-system/tasks/visual-diff/index.js

```diff
@@ -414,81 +387,88 @@ async function generateSnapshots(percy, webpages) {
 async function snapshotWebpages(percy, browser, webpages) {
   const pagePromises = {};
   for (const webpage of webpages) {
-    while (Object.keys(pagePromises).length >= MAX_PARALLEL_TABS) {
-      await sleep(WAIT_FOR_TABS_MS);
-    }
+    const {viewport, name: pageName} = webpage;
+    const fullUrl = `${BASE_URL}/${webpage.url}`;
+    for (const [testName, testFunction] of Object.entries(webpage.tests_)) {
```

Comment:
High level question: I assume the default flow is still to generate one snapshot per page (unless the page specifies one or more interactions)?